### ISSUE TITLE
Suggestion - move ingestion status into Data Tab

### ISSUE THREAD

**Issue by JohnAstronomer** [lsst-uk/lasair-lsst#143]
The Lasair ingestion status is currently under the support tab. To me it would be more useful & logical to be under the 'Data' tab. You may wish to check that the data status is Ok when testing or de-bugging queries or filters. One of the reasons filters or queries on recent data may fail is there is no recent data !

**Comment by RoyWilliams** [comment]
You are right. We should take the status information and put it on the front page!

**Comment by JohnAstronomer** [comment]
Sounds like the home page needs a status dashboard - over to you !
John
